lib/Github/HttpClient/Plugin/GithubExceptionThrower.php

```diff
@@ -59,7 +59,12 @@ public function doHandleRequest(RequestInterface $request, callable $next, calla
                 if (422 === $response->getStatusCode() && isset($content['errors'])) {
                     $errors = [];
                     foreach ($content['errors'] as $error) {
-                        switch ($error['code']) {
+                        $code = null;
+                        if (isset($error['code'])) {
+                            $code = $error['code'];
+                        }
+
+                        switch ($code) {
```

Comment:
We don't need the extra code above, this can be resolved in the switch line itself

```suggestion
                        switch ($error['code'] ?? null) {
```